packages/build/package.json

```diff
@@ -39,7 +39,6 @@
     "filesize": "^3.5.11",
     "hops-build-config": "10.3.0-rc.2",
     "hops-config": "10.3.0-rc.0",
-    "hops-express": "10.3.0-rc.0",
```

Comment:
This will be a breaking change - also *technically* all other changes in the peerDependency ranges are breaking.
So therefore we could also move hops-config and hops-build-config to peerDependencies and release this as hops@11.
/cc @dmbch